app/views/inquiries/index.html.erb

```diff
@@ -19,15 +19,15 @@
             method="get"
             role="search">
         <div class="form-group">
-          <input class="form-control"
+          <input class="form-control search-input"
```

Comment:
If you're using BEM this should be a modifier class of the form-control pattern, so I would do something like `<input class="form-control form-control--search-input" ...`.